src/components/Snackbar/AptoSnackbar.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  AptoSnack,
+  AptoSnackbarContext,
+  AptoSnackbarContextInterface
+} from './AptoSnackbarContext';
+import { AptoSnackbar } from './AptoSnackbar';
+import { AptoButton } from '../Button';
+import { AptoAlertVariants } from '../Alert';
+
+let snackCounter = 0;
+
+class SnackbarExample extends React.Component<
+  {},
+  AptoSnackbarContextInterface
+> {
+  public showSnack = (
+    message: React.ReactNode,
+    type: AptoAlertVariants = 'info',
+    action: React.ReactNode | null = null
+  ) => {
+    const snack: AptoSnack = {
+      message,
+      action,
+      type,
+      id: snackCounter += 1
+    };
+    const oldSnacks: AptoSnack[] = this.state.snacks || [];
+    this.setState({ snacks: oldSnacks.concat(snack) });
+  };
+
+  public hideSnack = (removedSnack: AptoSnack) => {
```

Comment:
If we don't need more than the id, let's just require the id rather than the whole snack?

One bit at a time, ya know?